gpii/configs/production.with.logging.json

```diff
@@ -22,7 +22,7 @@
             }
         }
     },
-    "includes": [
+    "mergeConfigs": [
         "../node_modules/deviceReporter/configs/production.json",
         "../node_modules/flowManager/configs/development.json",
         "../node_modules/solutionsRegistry/configs/development.json",
```

Comment:
Cant comment on line below, but the flatmatchmaker URL should be `gpii.flatMatchMaker.config.development.json` if we decide to keep this